### PR TITLE
tkt36 Bootstrap to menus Fixed Info links

### PR DESCRIPTION
**Pull request by vricci518**
Ticket #36 
Added bootstrap to menu pages
Added takeout info to takeout pages
Fixed links on Info dropdowns

### CODE REVIEW

File: clmenu.php [hunk 1]
```diff
@@ -43,137 +43,143 @@
...
+				<div class="col col-xs-12 col-sm-12 col-md-8 col-lg-8">
+					<p><b>BAR-Bill CHICKEN WING PIZZA</b><br />
+						Wing Chicken, Mozzarella Cheese and Spicy Blue Cheese sauce.<br />
+						$10.95
+					</p>
+				</div>
+				<div class="col col-xs-12 col-sm-12 col-md-4 col-lg-4">
+					<img src="images/pepperonipizza.jpg" alt="chicken wings">
+				</div>
+				<div class="col col-xs-12 col-sm-12 col-md-8 col-lg-8">
```
Comment (margeschrec) [inline clmenu.php]: is it necessary to mark sm-xs, and md-lg if they are both the same? from what I understand you would just need to specify sm-12, and md-4.

File: eamenu.php [hunk 2]
```diff
@@ -43,132 +43,138 @@
...
-				<p><b>CHEF SALAD</b><br />
-					Spring mix, Tomatoes, Onions, Croutons and Mozzarella Cheese.<br />
-					Choose: Ranch, Honey Mustard, Creamy Balsamic, or Italian
-					$6.95
-				</p>
+			<div class="row">
+				<div class="col col-xs-12 col-sm-12 col-md-4 col-lg-4">
+					<img src="images/breadsticks.jpg" alt="chicken wings">
+				</div>
+				<div class="mcol col-xs-12 col-sm-12 col-md-8 col-lg-8">
```
Comment (margeschrec) [inline eamenu.php]: mcol class? typo?

File: contact.php [hunk 3]
```diff
@@ -42,111 +42,111 @@
...
-				<p><em>Clarence</em><br />
-					8326 Main Street<br />
-					Clarence, NY 14221<br />
-					(716) 710-9464
-				</p>
-			</div>
-		</footer>
-    
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+				if (!empty($_POST["name"])) {
```
Comment (margeschrec) [inline contact.php:71]: I will need to push up my revised form, I just moved the trim function, fyi

**Review by margeschrec — APPROVED** [review]
made some comments, I will need to push up a few small changes to the form today. otherwise looks good, thank you for fixing the drop down menus.

File: eamenu.php (hunk 2 above)
Comment (billythakidd04) [reply eamenu.php]: Good catch

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
One class looks off but good otherwise

File: clmenu.php (hunk 1 above)
Comment (vricci518) [reply clmenu.php]: Fixed clmenu and eamenu

File: eamenu.php (hunk 2 above)
Comment (vricci518) [reply eamenu.php]: fixes ea menu, eatakeout and cltakeout

**Review by margeschrec — APPROVED** [review]

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Line 184 of eatakeout.php has no matching opening div tag.

**Review by billythakidd04 — APPROVED** [review]

**Comment by vricci518** [comment]
#37  menus and info links fixed

**Pull request merged**